engines/mtropolis/detection_tables.h

```diff
@@ -637,7 +739,7 @@ static const MTropolisGameDescription gameDescriptions[] = {
 			{
 				{ "insel.exe",      0, "0e513dac9d2a0d7cfcdc670cab2a9bda", 757760 },
 				{ "ile_myst1.MPL",  0, "44ac2e944a61303f4028408165bafeb4", 24080377 },
-				{ "ILEMYST2.MPX",   0, "ecb1d2c8b99eb5b2ac6be55893b97f50", 283748531 },
+				{ "ile_myst2.MPX",  0, "ecb1d2c8b99eb5b2ac6be55893b97f50", 283748531 },
```

Comment:
Interesting. Was it incorrect before?

Reply:
The file name is `ile_myst2.MPX` in my copy. Maybe there is a copy where `ILEMYST2.MPX` is correct. Maybe that would be the correct file name for the Mac version, although I doubt it.
But considering that the checksum and size still match, and that the file name is `ile_myst2.MPX` in each other language version, I will hazard a guess that this detection entry was just faulty.

Reply:
I'm assuming the original was a mistake because the MPX filenames to load are determined by the project name in the MPL file, so if the name changed then the MPL hash wouldn't match.

Reply:
The "project name" from the MPL is indeed `ile_myst`, and thus the file `ile_myst2.mpx` is expected. This deviation would indeed be highly unusual.